api/src/entities/controllers/user.ts

```diff
@@ -88,3 +89,84 @@ export const loginUser = (req: Request, res: Response) => {
       });
     });
 };
+
+export const processOneTimeCode = (req: Request, res: Response) => {
+  let min = 100000
+  let max = 999999
+  let generatedCode = min + (Math.floor(Math.random() * (max - min)))
+  User.findOneAndUpdate({ email: req.body.email }, { oneTimeCode: generatedCode })
+    .exec()
+    .then(user => {
+      if (!user) {
+        return res.status(400).json({
+          message: 'Email is not registered',
+        })
+      }
+      const transporter = nodemailer.createTransport({
+        service: 'gmail',
+        auth: {
+          user: 'devtimr',
+          pass: envConfig.GMAIL_PASSWORD,
+        },
+      });
+      const mailOptions = {
+        from: 'devtimr@gmail.com',
+        to: req.body.email,
```

Comment:
Maybe `user.email` might be slightly better

Reply:
Okay